kythe/cxx/indexer/cxx/IndexerASTHooks.cc

```diff
@@ -3259,6 +3262,16 @@ bool IndexerASTVisitor::AddNameToStream(llvm::raw_string_ostream& Ostream,
   return true;
 }
 
+void IndexerASTVisitor::AssignUSR(const GraphObserver::NodeId& TargetNode,
```

Comment:
Can you add docs (here or in the .h) that specify when we do and when we don't assign usr? I'm pretty sure you already have the text in the PR description.

Reply:
Done